rules/repositories.bzl

```diff
@@ -35,8 +35,8 @@ def rules_ios_dependencies():
     _maybe(
         git_repository,
         name = "build_bazel_rules_swift",
-        commit = "2ebfc403ffdb428befd38e25e236ae181aa1a49b",
-        shallow_since = "1586818875 -0700",
+        commit = "657eda57a097980848dbb3c880b94faeddaa3cd1",
+        shallow_since = "1580525034 -0800",
```

Comment:
I would say let's go back to something we know works, and update when `build_bazel_rules_swift` is more stable related to latest changes?

Reply:
cc @gyfelton

Reply:
Hi @acecilia I am working with @segiddins to keep this repo update to date with `rules_swift` as possible. Do you know which commit in between 657e  and 2ebfc that breaks your side?

Reply:
See https://github.com/bazel-ios/rules_ios/pull/53 for what I am working on right now

Reply:
Hey @gyfelton :) I am keeping an eye on your PR for some days now, eager for the fix :D My problem is similar to the one you mentioned there. If you take my PR and update the version of the swift rules, and then run:
```
bazel build --apple_platform_type=ios //tests/mixed-source/ios/only_sources/...
```
You will be getting the following errors:

```
ERROR: .../rules_ios/tests/mixed-source/ios/only_sources/BUILD.bazel:10:1: Compiling Swift module MixedSourceFramework failed (Exit 1)
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary/SwiftLibrary.framework/Modules/module.modulemap:1:18: error: redefinition of module 'SwiftLibrary'
framework module SwiftLibrary {
                 ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary.extended.modulemap:1:18: note: previously defined here
framework module SwiftLibrary {
                 ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary/SwiftLibrary.framework/Modules/module.modulemap:8:21: error: redefinition of module 'Swift'
module SwiftLibrary.Swift {
                    ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary.extended.modulemap:8:21: note: previously defined here
module SwiftLibrary.Swift {
                    ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary/SwiftLibrary.framework/Modules/module.modulemap:1:18: error: redefinition of module 'SwiftLibrary'
framework module SwiftLibrary {
                 ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary.extended.modulemap:1:18: note: previously defined here
framework module SwiftLibrary {
                 ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary/SwiftLibrary.framework/Modules/module.modulemap:8:21: error: redefinition of module 'Swift'
module SwiftLibrary.Swift {
                    ^
/private/var/tmp/_bazel_user/8cd744a626f0e61ccbc6ce5d2c80f1da/execroot/build_bazel_rules_ios/bazel-out/apl-ios_x86_64-fastbuild/bin/tests/mixed-source/ios/only_sources/SwiftLibrary.extended.modulemap:8:21: note: previously defined here
module SwiftLibrary.Swift {
                    ^
<unknown>:0: error: could not build Objective-C module 'SwiftShims'
```

If you do not have an inconvenience, I think it would be good to stay with a version that we know works well until we figure out the new changes in the swift rules

Reply:
Thanks for the detailed reply! My bad for not following up on this issue.
It seems that your side is pointing at the HEAD of master branch of `rules_ios`? Based of the error you put, I feel that this PR can fix it plus pointing at a later version of `rules_swift` (still not the latest because of some more breaking changes on their side)

In terms of stay with a version, wondering if pointing to a specific commit of master branch is a better idea for you side so that we can keep making progress while keep introducing more tests that would pick up breaking changes in future (like what I did in this PR that will fail if the changes are not in).

I am going to merge once green since I got approval from Sam. We can then discuss whether to revert it by this PR.

Reply:
I do not fully understand your message :D But what I got is that you are going to merge https://github.com/bazel-ios/rules_ios/pull/53. Thats okey, go ahead and I will rebase and test if the changes in this PR are needed after that

Reply:
ahhh my bad, I need to learn from Sam about whats the strategy of merging changes given this repo is shared by multiple ppl.
Side question: I saw you passing `--apple_platform_type=ios` as argument during build/test. I tried locally on my branch and i got error while compiling `rules/hmap` and the error is related to `invalid version number in '-mmacosx-version-min=13.2'` do you have the same error?

Reply:
Yes I do. I added a comment in this PR about it: https://github.com/bazel-ios/rules_ios/pull/58#discussion_r422545277

Reply:
@gyfelton I just rebased and pushed. The issues we talked about before are gone, I am happily using latest swift rules :)